intel_extension_for_transformers/neural_chat/pipeline/plugins/audio/utils/english_normalizer.py

```diff
@@ -50,10 +50,10 @@ def __init__(self):
         }
         
     def correct_abbreviation(self, text):
-        # if one word is all capital letters, then correct this whole word
```

Comment:
do you think it's good to maintain a keyword map? I guess it's hard to use reg exp to cover all the cases.